bin/cheritest/cheritest_vm.c

```diff
@@ -44,6 +44,7 @@
 #include <sys/time.h>
 #include <sys/ucontext.h>
 #include <sys/wait.h>
+#include <sys/mount.h>
```

Comment:
style(9) would generally prefer alphabetic sorting of header includes, where those headers permit it.